lib/main.js

```diff
@@ -37,8 +35,8 @@ import kernelManager from "./kernel-manager";
 import ZMQKernel from "./zmq-kernel";
 import WSKernel from "./ws-kernel";
 import Kernel from "./kernel";
-import AutocompleteProvider from "./autocomplete-provider";
 import HydrogenProvider from "./plugin-api/hydrogen-provider";
+import services from "./services";
```

Comment:
I think adding a short README.md in `./services` explaining what type of thing belongs there would be really useful to both to new contributors and maintainers.

Reply:
Good Idea